rdf_differ/entrypoints/diffs.py

```diff
@@ -0,0 +1,79 @@
+"""
+diffs.py
+Date: 30/07/2020
+Author: Mihai Coșleț
+Email: coslet.mihai@gmail.com
+"""
+from SPARQLWrapper.SPARQLExceptions import EndPointNotFound
+from werkzeug.datastructures import FileStorage
+
+from rdf_differ.diff_adapter import FusekiDiffAdapter
+from rdf_differ.skos_history_wrapper import SKOSHistoryRunner
+from utils.file_utils import temporarily_save_files
+
+
+def get_diffs() -> tuple:
+    """
+        List the existent datasets with their descriptions.
+    :return: list of existent datasets
+    :rtype: list, int
+    """
+    fuseki_adapter = FusekiDiffAdapter('http://localhost:3030')
```

Comment:
the environment variable must be used. 
It would not be entirely correct for Fuseki but we could use the ENDPOINT